### DISCUSSION TITLE
GitHub Copilot

### DISCUSSION THREAD

**Discussion by 13223926092** [community/community#16681] [Copilot Conversations] [answered]
I need to try out GitHub Copilot, but no email was sent to me

**Comment by 13223926092** [comment]
The application has been submitted for trial, and there is no feedback result！

**Comment by davecheney** [comment, marked as answer]
I'm sorry that you have not been given access yet. Please see https://github.com/github/feedback/discussions/9342#discussioncomment-1902060
